src/selectors/entities/channels.js

```diff
@@ -419,6 +419,14 @@ export const getDefaultChannel: (GlobalState) => ?Channel = createSelector(
     }
 );
 
+export const getDefaultChannelForTeam: (GlobalState, string) => ?Channel = createSelector(
```

Comment:
Same comment here about not using `createSelector` since this the selector has a parameter.

Reply:
Done